test/unit/remote-config/remote-config-api-client.spec.ts

```diff
@@ -20,13 +20,13 @@ import * as _ from 'lodash';
 import * as chai from 'chai';
 import * as sinon from 'sinon';
 import {
-  RemoteConfigApiClient,
   RemoteConfigTemplate,
   TagColor,
   ListVersionsResult,
   Version,
 } from '../../../src/remote-config/remote-config-api-client';
-import { FirebaseRemoteConfigError } from '../../../src/remote-config/remote-config-utils';
+import { RemoteConfigApiClient } from '../../../src/remote-config/remote-config-api-client-internal';
+import { FirebaseRemoteConfigError } from '../../../src/remote-config/remote-config-utils-internal';
```

Comment:
Just `remote-config-internal`. Or just move into the `remote-config-api-client-internal` module. `*-utils-internal` is an unusual name.

Reply:
Yeah. `remote-config-utils` only contains `FirebaseRemoteConfigError` and error codes. We should be able to move it into `remote-config-api-client-internal`